sunpy/map/mapbase.py

```diff
@@ -2364,11 +2364,11 @@ def peek(self, draw_limb=False, draw_grid=False,
         return figure
 
     @u.quantity_input
-    def plot(self, annotate=True, axes=None, title=True,
+    def plot(self, annotate=True, axes=None, title=True, autoalign=False,
```

Comment:
Should this be set to `None` by default, which doesn't autoalign and raises a warning (on different WCS), and then if the user manually specifies `False` the warning can be disabled as they've made a deliberate choice to not align the maps.

Reply:
I'm not keen on this suggestion.  We didn't have a way to disable the warning prior to the PR, and I don't know why we'd want to provide users with a way to turn off the warning.  If a user really wants to disable a warning, there's always `filterwarnings()`.